markdown/api-docs/cart-and-checkout/embedded-checkout.md

```diff
@@ -0,0 +1,335 @@
+{
+  "cart_url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?action=load&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b",
+  "checkout_url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?action=loadInCheckout&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b",
+  "embedded_checkout_url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?embedded=1&action=loadInCheckout&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b"
+}
+
+```
+
+Use the `embedded_checkout_url` that is returned and assemble a JSON object that will be used by the Checkout JS SDK to determine how to render the checkout.
+
+Read more about the [JSON object](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/README.md#embedcheckout) and its possible corresponding [rendering options](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/embeddedcheckoutoptions.md).
+
+Example of the JSON object:
+
+<!--
+title: "JSON object"
+subtitle: ""
+lineNumbers: true
+-->
+
+**JSON Object**
+
+```js
+{
+"containerId": "bc-embedded-checkout",
+"url": "https://store-id30h7ohwf.mybigcommerce.com/cart.php?embedded=1&action=loadInCheckout&id=bc218c65-7a32-4ab7-8082-68730c074d02&token=aa958e2b7922035bf3339215d95d145ebd9193deb36ae847caa780aa2e003e4b",
+"styles": {
+    "button": {
+	"backgroundColor": "#ffffff",
+	"borderColor": "#000000",
+	"color": "#000000"
+        }
+    }
+}
+
+```
+
+Once you have created the JSON object, in your app project folder navigate to `public > index.html`. Create an empty `<div>` with the id attribute mapped to the value for `containerId` in the page where you want to embed the checkout.
+
+Example: `<div id="bc-embedded-checkout"></div>`
+
+Finally, we will render the checkout from your app. Import and invoke the checkout service, and import and call the `embedCheckout` method while passing in your JSON object.
+
+<!--
+title: "External app"
+subtitle: ""
+lineNumbers: true
+-->
+
+**App.js**
+
+```js
+import React from 'react';
+import './App.css';
+import {createCheckoutService} from '@bigcommerce/checkout-sdk'
+import {embedCheckout} from '@bigcommerce/checkout-sdk'
+
+const service = createCheckoutService();
+
+
+class App extends React.Component {
+componentDidMount(){
+  embedCheckout({
+    "containerId": "bc-embedded-checkout",
+    "url": "https://my-site.com/cart.php?embedded=1&action=loadInCheckout&id=9a988ee4-be1e-4fbd-9e3c-5cc32df42cbd&token=bccc9a5e9691a7d30267a3188c238d7a22d2d9685734e926cf4c261f7c83de13",
+    "styles": {
+      "button": {
+        "backgroundColor": "#ffffff",
+        "borderColor": "#000000",
+        "color": "#000000"
+      }
+    }
+  })
+ }
+ render(){
+   return <h1>BC Embedded Checkout</h1>
+ }
+}
+
+export default App;
+```
+
+At this point, you should have a working embedded checkout. 
```

Comment:
Anything else you have to do? Restart the app or navigate to a certain page? What about Heroku--where does that come in?